tests/cypress/integration/digitall.home.spec.ts

```diff
@@ -1,55 +0,0 @@
-import { siteHomePage } from '../page-object/site.home.page'
```

Comment:
Why are we removing this test?

Reply:
Because it's redundant, it's testing the same thing mysite test it testing.

Reply:
OK